### DISCUSSION TITLE
How many kernels will you find on an average ear of 🌽?

### DISCUSSION THREAD

**Discussion by ParanoidUser** [ParanoidUser/yolo#912] [Galaxy Trivia] [answered]
*Answer in baker's dozens.*

**Comment by peterbaccaro** [comment, marked as answer]
On an average ear of corn, you will typically find around 600 to 800 kernels. The exact number can vary depending on the variety of corn and growing conditions.

A baker's dozens is 13 so to calculate in bakers dozen the numbers need to be divided by 13.

600 ÷ 13 = 46 baker's dozens + 2
800 ÷ 13 = 61 baker's dozens + 7

So in bakers dozens an average ear of corn will have **around 46 to 61 bakers dozens of kernels**.

**Comment by raeef1001** [comment]
On an average ear of corn, you will typically find around 600 to 800 kernels. The exact number can vary depending on the variety of corn and growing conditions.

A baker's dozens is 13 so to calculate in bakers dozen the numbers need to be divided by 13.

600 ÷ 13 = 46 baker's dozens + 2
800 ÷ 13 = 61 baker's dozens + 7

So in bakers dozens an average ear of corn will have around 46 to 61 bakers dozens of kernels.
